src/app/public/modules/autocomplete/autocomplete.component.ts

```diff
@@ -153,6 +156,7 @@ export class SkyAutocompleteComponent
   private _debounceTime: number;
   private _descriptorProperty: string;
   private _dropdownController = new Subject<SkyDropdownMessage>();
+  private _dropdownIsOpen: boolean  = false;
```

Comment:
I would rename this to `isDropdownOpen`. We use the underscore for properties that are associated with getters and setters.

Reply:
Ah, gotcha. Went ahead and updated it!